okhttp/src/jvmMain/kotlin/okhttp3/ConnectionSpec.kt

```diff
@@ -51,6 +51,15 @@ class ConnectionSpec internal constructor(
   private val tlsVersionsAsString: Array<String>?
 ) {
 
+  /**
+   * Returns the cipher suites to use for a connection. Returns null if all of the SSL socket's
+   * enabled cipher suites should be used.
+   */
+  @get:JvmName("cipherSuitesAsString") val cipherSuitesAsStringList: List<String>?
```

Comment:
This is new public api, which seems unfortunate.

From kotlin

```
connectionSpec.cipherSuitesAsStringList
```

From java

```
tlsSpec.cipherSuitesAsString()
```